components/bear-market-airdrop/Blocks/Hero/Email.tsx

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable no-restricted-imports */
```

Comment:
don't ignore this instead read what the errors say, we have <Text> as an import from `tw-components` which comes with out default text styling out of the box for example